tests/unit/data/datasets/test_preference_dataset.py

```
@@ -0,0 +1,239 @@
+        },
+    ]
+
+    val_data = [
+        {
+            "prompt": "What is 3*3?",
+            "chosen_response": "The answer is 9.",
+            "rejected_response": "The answer is 6.",
+        }
+    ]
+
+    train_ctx = tempfile.NamedTemporaryFile(mode="w", suffix=".json", delete=False)
```

Ruff (0.12.2)

Local variable `train_ctx` is assigned to but never used

Remove assignment to unused variable `train_ctx`

(F841)